packages/flutter/test/widgets/resizing_header_sliver_test.dart

```diff
@@ -0,0 +1,122 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+
+void main() {
+  testWidgets('ResizingHeaderSliver basics', (WidgetTester tester) async {
```

Comment:
Similar comment to the others, there should be a test for each permutation of [vertical, horizontal] : [reversed, not].

Reply:
Will do.